interfaces/builtin/utils_test.go

```diff
@@ -70,3 +73,19 @@ func MockPlug(c *C, yaml string, si *snap.SideInfo, plugName string) *interfaces
 func MockSlot(c *C, yaml string, si *snap.SideInfo, slotName string) *interfaces.Slot {
 	return builtin.MockSlot(c, yaml, si, slotName)
 }
+
+func MockConnectedPlug(c *C, yaml string, si *snap.SideInfo, plugName string) (*interfaces.ConnectedPlug, *snap.PlugInfo) {
```

Comment:
Can we reuse those from export test?

Reply:
No, not really, not easily at least. I once looked at it as we have other duplicated stuff there. They are in two separate packages (builtin and builtin_test) and we have tests depending on one or the other. Once I attempted to get rid of these duplicated helpers I got into a rabbit hole. Would be worth investigating at some point though.